doc/ansible/dev/advanced_options.md

```diff
@@ -41,7 +41,11 @@ Some features can be overridden per resource via an annotation on that CR. The o
 
 The ansible runner will keep information about the ansible run in the container.  This is located `/tmp/ansible-operator/runner/<group>/<version>/<kind>/<namespace>/<name>`. To learn more  about the runner directory you can read the [ansible-runner docs](https://ansible-runner.readthedocs.io/en/latest/index.html).
 
-## Turning off Dependent Watches and Owner Reference Injection
+## Owner Reference Injection
+
+Owner references enable [Kubernetes Garbage Collection](https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/) to clean up after a CR is deleted. Owner references are injected by ansible operators by default by the proxy.
+
+Owner references only apply to resources in the same namespace as the CR. Resources outside the namespace of the CR will automatically be annotated with `operator-sdk/primary-resource` and `operator-sdk/primary-resource-type` to track creation. These resources will not be automatically garbage collected. To handle deletion of these resources, use a [finalizer](finalizers.md).
```

Comment:
May it should be after the next paragraph, wdyt?

Reply:
I thought it made more sense to explain what happens, and then explain how you can change it.